packages/cursorless-engine/src/testCaseRecorder/RecordTestCaseCommandArg.ts

```diff
@@ -0,0 +1,52 @@
+import { ExtraSnapshotField } from "@cursorless/common";
+
+/**
+ * This is the type of the `recordTestCase` command's argument. It is used to
+ * specify the options for recording a test case.
+ */
+export interface RecordTestCaseCommandArg {
```

Comment:
I've seen these referred to as "`options` objects", which I typically shorten to `opts`. https://stackoverflow.com/questions/32948475/argument-vs-options-in-typescript

Perhaps options would be a better name than `arg` if it's being exposed?